lesson_06/homeworks_06.py

```diff
@@ -21,12 +33,30 @@
 Якщо прибулець червоний, надрукуйте повідомлення про те, що гравець заробив 15 очок
 + напишіть цикл for що перебере і обробить всі значення списку alien_color
 """
+print("\ntask 03, 04")
+alien_color = ('green', 'yellow', 'red')
+for color in alien_color:
+    if color == "green":
+        print("Ви щойно збили прибульця зеленого кольору і заробили +5 балів!")
+    elif color == 'red':
+        print("Ви збили прибульця червоного кольору і заробили +15 балів!")
+    elif color != 'green':
+        print("Ви збили прибульця не зеленого кольору і заробили +10 балів!")
 
 # task 5
 """  Начинки для піци (pizza_topping): напишіть цикл, який пропонує користувачеві ввести ряд начинок
 для піци, доки він не введе значення 'quit'. Коли вони введуть кожну начинку,
 надрукуйте повідомлення про те, що ви додасте цю начинку до їхньої піци.
 """
+print("\ntask 05")
+pizza_topping = f'гриби, бекон, пармезан, перець, оливки, хамон, груша, горгонзолла'
+print(f'Пропонуємо наступні начинки на вибір: {pizza_topping}')
+for topping_name in pizza_topping:
+    topping_name = input("Введіть бажану назву начинки із запропонованих, або для завершення введіть quit: ").lower()
+    if topping_name.lower() == 'quit':
+            break
```

Comment:
```suggestion
        break
```
щось тут відступ завеликий
не помилка, але може спричинити помилку